qiita_db/metadata_template/test/test_util.py

```diff
@@ -147,12 +147,23 @@ def test_load_template_to_dataframe_lowercase(self):
         exp.rename(columns={"str_column": "str_CoLumn"}, inplace=True)
         assert_frame_equal(obs, exp)
 
-    def test_load_template_to_dataframe_non_utf8(self):
+    def test_load_template_to_dataframe_non_utf8_error(self):
         bad = EXP_SAMPLE_TEMPLATE.replace('Test Sample 2', 'Test Sample\x962')
         with self.assertRaises(ValueError):
             qdb.metadata_template.util.load_template_to_dataframe(
                 StringIO(bad))
 
+    def test_load_template_to_dataframe_non_utf8(self):
+        replace = EXP_SAMPLE_TEMPLATE.replace(
+            'Test Sample 2', u'Test Sample\x962')
+        qdb.metadata_template.util.load_template_to_dataframe(
+            StringIO(replace))
+        # setting back
+        replace = EXP_SAMPLE_TEMPLATE.replace(
+            u'Test Sample\x962', 'Test Sample 2')
+        qdb.metadata_template.util.load_template_to_dataframe(
+            StringIO(replace))
+
```

Comment:
Unless I am missing something obvious, this test is missing a "test". Is "not erroring" the test? If so, are there any checks that can be done on the returned value from `load_template_to_dataframe`?

Reply:
The test is that it not raises an error, AKA that it can be done `test_load_template_to_dataframe_non_utf8_error` has the test for the raise.

Reply:
Ah I see, thanks! 👍